### PR TITLE
fix(docs): update Script stage documentation.

### PR DESCRIPTION
**Pull request by avh121**

### CODE REVIEW

**Review by lwander — APPROVED** [review]
LGTM, but I know nothing about this stage :sweat: -- was @ezimanyi helping you troubleshoot? Maybe he can have a look.

File: setup/ci/jenkins.md [hunk 1]
```diff
@@ -91,56 +91,73 @@ b. Under __Crumb Algorithm__, select __Default Crumb Issuer__.
...
 
-## Jenkins script execution stage
+## Configure script stage
 
 ### Purpose
-The script stage lets a Spinnaker user run an arbitrary shell, python, or
-groovy script on a Jenkins instance as a first class stage in Spinnaker.
-This is good for launching an integration/functional test battery
+The [Script stage](/reference/pipeline/stages/#script) lets you run an arbitrary
+shell, python, or groovy script on a Jenkins instance as a first class stage in
```
Comment (dorbin) [inline setup/ci/jenkins.md]: Captialize Python and Groovy?

File: setup/ci/jenkins.md [hunk 2]
```diff
@@ -91,56 +91,73 @@ b. Under __Crumb Algorithm__, select __Default Crumb Issuer__.
...
-Here are a few assumptions we make in the following directions:
+### Prerequisites
 
-You have a running Spinnaker instance, with access to configuration files.
+In order to configure a Script stage, you need:
 
-You have a running Jenkins instance at `<jenkins_host>`, with a user profile set up with admin access.
+*   A running Spinnaker instance, with access to configuration files.
+*   A running Jenkins instance at `$JENKINS_HOST`, with a user profile set up
+    with admin access.
```
Comment (dorbin) [inline setup/ci/jenkins.md]: No need for periods at the ends of these

File: setup/ci/jenkins.md [hunk 3]
```diff
@@ -91,56 +91,73 @@ b. Under __Crumb Algorithm__, select __Default Crumb Issuer__.
...
 
-At this point, you should be able to manually run the script job in Jenkins
-(with parameters) and see it succeed.
+    1.  Add the GitHub repository containing your scripts.
+    2.  Either create a `Spinnaker` node in which Jenkins will run all Script
+        jobs, or de-select the **Restrict where this project can be run**
+        checkbox.
 
-### Summary
+    At this point, you should be able to manually run the script job in Jenkins
```
Comment (dorbin) [inline setup/ci/jenkins.md]: s/should be able to/can/ 
Or else, under what conditions might I not be able to?

File: setup/ci/jenkins.md [hunk 4]
```diff
@@ -91,56 +91,73 @@ b. Under __Crumb Algorithm__, select __Default Crumb Issuer__.
...
+    script:
+      master: your-jenkins-master
+      job: $JOB_NAME  # from step #3
+    ```
 
-The current version of the script stage is a bit rudimentary, but we'll
-soon have support for a separate "job" stage that will be much more robust and encapsulate
-the same behavior as the script stage. The script stage is a temporary
-solution for the time being.
+You should now be able to use the Script stage in your pipelines.
```
Comment (dorbin) [inline setup/ci/jenkins.md]: Same comment here, re "should."

**Review by dorbin — COMMENTED** [review]
A few editorial nits, otherwise LGTM

File: setup/ci/jenkins.md [hunk 5]
```diff
@@ -91,56 +91,73 @@ b. Under __Crumb Algorithm__, select __Default Crumb Issuer__.
...
-## Jenkins script execution stage
+## Configure script stage
 
 ### Purpose
-The script stage lets a Spinnaker user run an arbitrary shell, python, or
-groovy script on a Jenkins instance as a first class stage in Spinnaker.
-This is good for launching an integration/functional test battery
+The [Script stage](/reference/pipeline/stages/#script) lets you run an arbitrary
+shell, Python, or Groovy script on a Jenkins instance as a first class stage in
+Spinnaker. This is good for launching an integration/functional test battery
```
Comment (ezimanyi) [inline setup/ci/jenkins.md]: Really minor (and pre-existing), but the term "battery" sticks out to me here...maybe "test suite" instead?  (I've never seen the word "battery" in the sense of "many similar things" except in the form "battery of", ex "battery of tests").

**Review by ezimanyi — APPROVED** [review]

**Pull request merged**